.github/workflows/deploy.yml

```diff
@@ -9,31 +9,41 @@ jobs:
   docs-deploy:
     name: Deploy docs to webpage
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [16.13]
     steps:
+      - name: Checkout
+        uses: actions/checkout@v2.3.4
+
+      - name: Pnpm
+        uses: pnpm/action-setup@v2.2.1
+        with:
+          version: 6.32.3
```

Comment:
why we use specifically pnpm? what are advanteges over npm?

Reply:
pnpm is up to 2x faster than npm, and it' s node_modules folder takes far less space than npm.

That was a personal preference, but if you want to stick with npm, I'll make the necessary changes